pie/help.py

```diff
@@ -33,7 +33,11 @@ async def acl_check(self, cmd: Union[commands.Group, commands.Command]) -> bool:
         bot = cmd._cog.bot
         ctx = self.context
         command = cmd.qualified_name
-        allow_invoke: Optional[bool] = acl.can_invoke_command(bot, ctx, command)
+        bot_owner_ids: Set = getattr(bot, "owner_ids", {*()})
+
+        allow_invoke: Optional[
+            bool
+        ] = ctx.author.id in bot_owner_ids or acl.can_invoke_command(bot, ctx, command)
```

Comment:
I'm just nitpicking, but I have to ask for this.

This formatting is ugly, a bit prettier would be shifting the whole second argument below:
```py
allow_invoke: Optional[bool] = ctx.author.id in bot_owner_ids or (
    acl.can_invoke_command(bot, ctx, command)
)
```

Reply:
OK, changed.